src/screens/SearchScreen.js

```diff
@@ -17,13 +17,17 @@ import {map, navKeys} from '../constants';
 
 class SearchScreen extends Component {
     componentWillMount() {
-        this.props.initializeMap();
+        const {region, initializeMap} = this.props;
+
+        initializeMap(region);
```

Comment:
Same pattern here in `SearchScreen` that we saw in `LoginScreen`; if a `region` already exists from a successful rehydration, then use it. If not, then the reducers down the line will populate `region` with the initial defaults.